desktop/assets/texts/uk-UA/main_menu.ftl

```diff
@@ -36,3 +36,5 @@ debug-menu-open-movie = Переглянути фільм
 debug-menu-open-movie-list = Показати відомі фільми
 debug-menu-open-domain-list = Показати домени
 debug-menu-search-display-objects = Пошук зображуваних об’єктів...
+view-menu = Переглянути
```

Comment:
That doesn't seem right, as it's not a noun (https://en.wiktionary.org/wiki/%D0%BF%D0%B5%D1%80%D0%B5%D0%B3%D0%BB%D1%8F%D0%BD%D1%83%D1%82%D0%B8)

Reply:
I flagged this on Crowdin

Reply:
I've deleted it for now, and re-downloaded the current translations.